hunt/app/views/animals/index.html.erb

```diff
@@ -1,18 +1,23 @@
 <ul>
-  <% @animals.each do |animal| %>
 
-    <div class="container">
-      <div class="row">
-        <div class="col-sm">
-          <li>Name: <%= link_to animal.name, animal_path(animal)  %></li>
-        </div>
-        <div class="col-sm">
-          State: <%=animal.state%>
-        </div>
-        <div class="col-sm">
-          Category: <%=animal.category %>
-        </div>
-      </div>
-  </div>
-  <% end %>
+  <div class="container">
+    <%= link_to 'New Animal', new_animal_path, class: 'btn btn-dark' %>
+<table class="table table-hover">
```

Comment:
This table looks great! Do you think you could fix the indentation? Also, remember that all the page content should go inside a container/row/column.